content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -23,6 +23,20 @@ in an old version of longhorn-instance-manager until it is detached (by scaling
 reattached (by scaling its consuming workload up). Consider scaling workloads down and back up again as soon as possible
 after upgrading from a version without this mechanism (v1.5.1 or older) to v{{< current-version >}}.
 
+### Default Priority Class
+
+Longhorn v1.6.0 introduces the default Priority Class `longhorn-critical`, which has the highest value and ensures that Longhorn pods are not evicted by kube-scheduler when system resources are low.
+
+During upgrades, Longhorn applies the default Priority Class to components depending on specific settings.
+
+- When all volumes are detached and the Priority Class default setting is not customized, the default Priority Class is applied to all Longhorn components. The default Priority Class setting `priority-class` is updated.
+- When all volumes are detached and you have specified a value for the Priority Class default setting, the default Priority Class is applied only to user-deployed components. The default Priority Class setting `priority-class` is not updated.
+- When one or more volumes are attached, the default Priority Class is applied only to user-deployed components only if they haven't been set. The default Priority Class setting `priority-class` is not updated.
```

Comment:
> applied only to user-deployed components only if they haven't been set

What do you mean by "if they haven't been set"?

Reply:
>"if they haven't been set"

It means that if users do not set the `priorityClass` in user-managed components such as https://github.com/longhorn/longhorn/blob/master/chart/values.yaml#L351-L356, the default Priority Class `longhorn-critical` would be set to the `prioriClass`.